lazer/sdk/proto/publisher_update.proto

```diff
@@ -0,0 +1,32 @@
+syntax = "proto3";
+
+package pyth_lazer_transaction;
```

Comment:
Please specify `optional` for all non-repeated fields, as recommended [here](https://protobuf.dev/programming-guides/proto3/#field-labels). Implicit fields are still optional but result in codegen that prevents detection of missing fields when decoding. For every optional field that's really optional, we should also document when it's correct to actually omit it (e.g. for prices where it no known value). And we can add a comment at the top that every field is actually required unless documented otherwise.